src/backend/fts/ftsmessagehandler.c

```diff
@@ -448,8 +453,10 @@ HandleFtsWalRepPromote(void)
 			 " DBState = %d", state);
 	}
 
+#if defined(FAULT_INJECTOR) || !defined(USE_INTERNAL_FTS)
```

Comment:
1. FAULT_INJECTOR should not be here. It has no related code here.
2. USE_INTERNAL_FTS should be level up. The whole function implements internal fts, not only a piece of code.

Reply:
This code will be compiled only if both conditions are met. Otherwise a compilation error 'defined but not used' will occur.